rep-2011.rst

```diff
@@ -877,6 +877,22 @@ Similar to the integration with launch, you could run ``ros2 bag play ...`` and
 
 That option would handle the conversion on the fly using the same mechanisms as the command line tool.
 
+Security Concerns
+=================
+
+The additional features this REP proposes do not impose significantly increased security concerns, since the message conversions happen over topics and services.
```

Comment:
I like where this is going but I would suggest rephrasing this a bit and avoiding strong claims. After all, we don't know if an unintended bug in the implementation of these features (or any of its dependencies, if any) would actually impose a significant security concern. In my experience security is a process, strong claims do not generally hold for long.

Here's a suggestion:
```suggestion
The additional features this REP proposes happen over topics and services, as any ROS 2 application would. Correspondingly, the use of evolving message types shouldn't add significant additional complexity to security thread models however new ROS 2 abstractions will be created when using it, and these would need to be secured appropriately. To that end, this REP makes the following security considerations:
```